src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -600,20 +600,20 @@ public static class TextCatalog
         public static LatentDirichletAllocationEstimator LatentDirichletAllocation(this TransformsCatalog.TextTransforms catalog,
```

Comment:
>LatentDirichletAllocation [](start = 57, length = 25)

Whole text catalog is slightly different from any other catalogs.
In any other catalog, we expose constructor with few major options, and then just columnOptions array.
In this specific case we expose EVERYTHING.
Shall we cut it to bare minimum?

Reply:
Ok.

---
In reply to: [263974378](https://github.com/dotnet/machinelearning/pull/2890#discussion_r263974378) [](ancestors = 263974378)